helm_deploy/values-dev.yaml

```diff
@@ -5,6 +5,7 @@ generic-service:
   replicaCount: 2
 
   ingress:
+    enable_allow_list: false
     host: create-and-vary-a-licence-dev.hmpps.service.justice.gov.uk
```

Comment:
Experimental. I don't think this is part of the `generic-service` helm chart but trying anyway. If this does not work, I'll comment the allow list entries and add one, over-arching allow-all rule.